### ISSUE TITLE
Broadcast Tx "send" button missing

### ISSUE THREAD

**Issue by robertandrewsmith** [MyCryptoHQ/MyCrypto#2480]
I recently updated to Windows version 1.6.3 to create and sign a tx offline, then broadcast online.

The "send" button is missing on the Broadcast Tx page.

Please Fix.

---

### Description of Issue

...

### What exactly were you doing when you encountered this issue?

...Give us some steps to reproduce...

### How were you accessing your wallet?

...Ledger, Trezor, MetaMask, Private Key, Keystore File, Parity Signer, Mnemonic Phrase?...

### Description of Your Machine

- Browser:
- Browser version:
- Operating system:
- Device:
- MyCrypto.com vs Desktop Application?
- MyCrypto Version Number:

### Console Logs / Screenshots

...

**Comment by tayvano** [comment]
Let's discuss and solve in #2481

**Comment by robertandrewsmith** [comment]
ok, so it appears that I pasted non-address text into the broadcast tx field, which is the cause of this issue. I did not notice initially because the  design of the field pushes my view of the text to the end of the line...not the beginning of the line where I had accidentally copy/pasted the bad text.
I think it's better to have a field that is several lines of space so the user can see all of their text. this should be an easy-peasy fix. ;-)
